spec/lucky/text_response_spec.cr

```diff
@@ -30,13 +30,29 @@ describe Lucky::TextResponse do
       context.request.body.to_s.should eq ""
       context.response.status_code.should eq 200
     end
+
+    it "gzips if enabled" do
+      Lucky::Server.temp_config(gzip_enabled: true) do
+        output = IO::Memory.new
+        context = build_context_with_io(output)
+        context.request.headers["Accept-Encoding"] = "gzip"
+        expected_io = IO::Memory.new
+        Gzip::Writer.open(expected_io) { |gzw| gzw.print "some body" }
```

Comment:
Nicely handled 👍 One suggestion I have is to move this to verify phase on line 45 so it is closer to where it is used